ui/src/cohort.ts

```diff
@@ -68,12 +68,20 @@ function generateFilter(group: Group): tanagra.Filter | null {
 // otherwise opaque except for a set of common methods.
 export interface Criteria {
   id: string;
-  type: string;
+  config: CriteriaConfig;
   name: string;
   count: number;
   data: unknown;
 }
 
+// CriteriaConfigs are used to initialize CriteriaPlugins and provide a list of
+// possible criteria.
+export interface CriteriaConfig {
+  type: string;
```

Comment:
Could you add some documentation specifically for this property? I'm not sure if this is a UI concept, a Tanagra API concept, or something else

Reply:
Done.